### PR TITLE
feat: add yat banner

### LINKED ISSUE #2326: Add YAT callout into the left nav
### Overview

As seen in this mock up https://www.figma.com/file/vm0jbvOrnLGj2ab8vmW1h7/Y.at-%3C%3E-Shapeshift-Web-App?node-id=875%3A4545

We are adding a YAT CTA that will open YAT in a new browser tab.

The CTA should change size as the screen size changes.

### References and additional details

n/a

### Acceptance Criteria

User can see and click on the YAT CTA in the left nav to get or login to their YAT account

### Need By Date

n/a

### Screenshots/Mockups

_No response_

### Estimated effort

_No response_

### PR DESCRIPTION
**Pull request by a-hirsch**
## Description

- Added Yat feature flag
- Added Yat banner to sidebar
- Added Yat banner to send modal

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

Closes #2326 

## Risk

None

### CODE REVIEW

File: src/components/Layout/Header/NavBar/YatBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            filter={`blur(${isBig ? 40 : 8}px)`}
+            position='absolute'
+            zIndex='hide'
+            bgImage='radial-gradient(circle at center,#7B61FF, transparent)'
+          />
+          <YatIcon w={isBig ? '29px' : '19px'} h={isBig ? '36px' : '24px'} />
+          {isBig && (
+            <Flex flexDir='column' gap='1'>
+              <Heading as='h2' size='24px' lineHeight='24px' fontWeight='semibold'>
+                Get a Yat
```
Comment (gomesalexandre) [inline src/components/Layout/Header/NavBar/YatBanner.tsx:51]: Translation

File: src/components/Layout/Header/NavBar/YatBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+              </Heading>
+              <Text
+                translation='Your Yat is your universal emoji username, website&nbsp;URL, payment address, and more.'
```
Comment (gomesalexandre) [inline src/components/Layout/Header/NavBar/YatBanner.tsx:54]: Add to translations file

File: src/components/Modals/Send/YatBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          h='100px'
+          filter='blur(40px)'
+          position='absolute'
+          zIndex='hide'
+          bgImage='radial-gradient(circle at center,#7B61FF, transparent)'
+        />
+        <YatIcon w='29px' h='36px' />
+        <Flex flexDir='column' gap='1.5'>
+          <Heading as='h2' size='24px' lineHeight='24px' fontWeight='semibold'>
+            Get a Yat
```
Comment (gomesalexandre) [inline src/components/Modals/Send/YatBanner.tsx:32]: Translation

**Review by gomesalexandre — CHANGES_REQUESTED** [review]
Initial comments, tested on various viewports and visually looks good overall, with no regressions with the flag off. Requested changes on missing translations.

Also, I'm wondering if @shapeshift/product agrees with this active style ?

image
image

File: src/assets/translations/en/main.json [hunk 4]
```diff
@@ -1315,5 +1315,13 @@
         "neverSellData": "Never sell data for profit"
       }
     }
+  },
+  "features": {
+    "yat": {
+      "banner": {
+        "title": "Get a Yat",
+        "description": "Your Yat is your universal emoji username, website&nbsp;URL, payment address, and more."
```
Comment (gomesalexandre) [inline src/assets/translations/en/main.json:1323]: This doesn't properly encode when used as a translation

image

What are we trying to achieve here? Looking at the mocks, we shouldn't need this. I get it why we would want "website URL" to always be displayed as one subject on all viewports, but it would mean splitting the translations which make globalization work more difficult. I'd simply remove it and use a regular space.

**Review by gomesalexandre — CHANGES_REQUESTED** [review]

File: src/components/Modals/Send/YatBanner.tsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+          filter='blur(40px)'
+          position='absolute'
+          zIndex='hide'
+          bgImage='radial-gradient(circle at center,#7B61FF, transparent)'
+        />
+        <YatIcon w='29px' h='36px' />
+        <Flex flexDir='column' gap='1.5'>
+          <Heading as='h2' size='24px' lineHeight='24px' fontWeight='semibold'>
+            {translate('features.yat.banner.title')}
+          </Heading>
```
Comment (gomesalexandre) [inline src/components/Modals/Send/YatBanner.tsx:35]: Note: `translate` will now be unused, it should be removed and you can now convert this component to the implicit return shorthand on l.6-8
```suggestion
          <Heading as='h2' size='24px' lineHeight='24px' fontWeight='semibold'>
            <Text translation='features.yat.banner.title' />
          </Heading>
```

**Review by gomesalexandre — CHANGES_REQUESTED** [review]
Smallish comments

Also, please refer to the answer from product WRT https://github.com/shapeshift/web/pull/2514#pullrequestreview-1080152277:

> we'd like the entire button to have a hover state

File: src/components/Layout/Header/NavBar/YatBanner.tsx [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+import { useTranslate } from 'react-polyglot'
+import { YatIcon } from 'components/Icons/YatIcon'
+import { Text } from 'components/Text'
+import { breakpoints } from 'theme/theme'
+
+type YatBannerProps = {
+  isCompact?: boolean
+}
+
+export const YatBanner = ({ isCompact }: YatBannerProps) => {
```
Comment (0xdef1cafe) [inline src/components/Layout/Header/NavBar/YatBanner.tsx:11]:
```suggestion
export const YatBanner: React.FC<YatBannerProps> = ({ isCompact }) => {
```

File: src/assets/translations/en/main.json [hunk 7]
```diff
@@ -1319,5 +1319,13 @@
         "neverSellData": "Never sell data for profit"
       }
     }
+  },
```
Comment (0xdef1cafe) [inline src/assets/translations/en/main.json:1322]: we only need to add the english translation - adding translations for the other languages actually makes it more difficult for @thesmithdao and @firebomb1 to know when things need updating.

can you please revert the changes to the other languages and just make these changes to the english files.

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]
two small requested changes, but overall implementation is looking nice

**Review by gomesalexandre — APPROVED** [review]
Approving this one so that we can merge it and @reallybeard can PR additional hover state improvements, as this is on another remote's branch.
